### DISCUSSION TITLE
Problem with Waves plugins

### DISCUSSION THREAD

**Discussion by LaZZco** [apohl79/audiogridder#1175] [Help]
I have a big problem with Waves plugins, everything is upside down
![Capture d’écran 2023-03-11 à 15 08 05](https://user-images.githubusercontent.com/127608960/224489691-836e6398-1d05-4970-acd6-42bfa70098a2.png)
<img width="1131" alt="Capture d’écran 2023-03-11 à 15 08 54" src="https://user-images.githubusercontent.com/127608960/224489697-7129f5f2-4939-4d04-8d94-40cda7016d8c.png">

**Comment by oldtos** [comment]
This seems to be a known issue with waves plugins and some ( many) DAWs.. hoops://forum.waves.com/t/plug-ins-gui-upside-down/2148/35

**Reply by jacko116** [reply]
Hello,
Happening to also to me. Tried instead Blue Cat's Patchwork with their Connector plugin and the issue does not occur.

What could be the problem?

**Comment by th3funnychurch3s** [comment]
As soon as this is fixed, I'm switching from Waves Soundgrid to Audio Gridder for running ALL plugins!

**Reply by jacko116** [reply]
A workaround, but not ideal, is running the plug-ins on the server side using Bluecat's Patchworks which shows the waves plugins GUI properly. So i would launch Audio Gridder, then an instance of Pachworks and then load the Waves VST.

**Comment by apohl79** [comment]
Logged a bug for this issue.

**Comment by th3funnychurch3s** [comment]
My only concern is this - what if my AG server becomes temporarily unavailable? E.g. Studiorack has the option to switch between Soundgrid and Native mode for such a scenario.

**Reply by apohl79** [reply]
If you had all your plugins installed locally as well (including licenses etc), you could run AG server locally and remotely and switch the entire session between them.

**Comment by Loupap** [comment]
Was there ever a fix for this upside down problem with Waves Plugins?

I am experiencing it with many of the waves plugins.
